src/test/kotlin/AgeTest.kt

```diff
@@ -0,0 +1,40 @@
+import kage.Age
+import kage.crypto.chacha20.ChaCha20Poly1305OutputStream
+import kage.crypto.scrypt.ScryptRecipient
+import kage.crypto.scrypt.ScryptRecipientTest
+import org.junit.Test
+import java.io.ByteArrayInputStream
+import java.io.ByteArrayOutputStream
+import java.io.FileOutputStream
+import java.nio.file.Files
+import java.util.*
+import kotlin.io.path.pathString
+import kotlin.test.assertEquals
+
+class AgeTest {
+    @Test
+    fun testEncryptDoesNotThrow() {
+        val recipients = listOf(ScryptRecipient("mypass1".toByteArray(), 18))
+
+        val bais = ByteArrayInputStream("this is my file".toByteArray())
+        val baos = ByteArrayOutputStream()
+
+        Age.encrypt(recipients, bais, baos, generateArmor = false)
+
+//        println(Base64.getEncoder().encodeToString(baos.toByteArray()))
+    }
```

Comment:
I just tested this didn't throw, as we didn't implement `decrypt` yet. I tried to shell out to the golang `age` binary to make sure this could be decrypted, but it requires a tty, so it was not possible. I cannot find a simple way to get around this. I could write a golang binary that uses `age` as a lib and decrypts a payload, but I think that should rather be tested in some kind of integration test?

In any case I manually tested if I could decrypt these payloads using `age` and it works.

Reply:
I don't think there's a point in adding any complicated logic to the test when we're still in WIP stage. Let's mark this test as `TODO` and we can revisit it later once the other things are working. 

Also, as for integration tests, we can definitely add them later.